### DISCUSSION TITLE
선행연구

### DISCUSSION THREAD

**Discussion by SHEYCHAE** [Pleasant-riot/Lung-Disease-Detection#1] [선행연구]
## Abstract
폐렴은 박테리아나 바이러스에 의한 호흡기 감염니다.
폐렴은 체액이 폐를 채우고 호흡곤란을 일으키는 상태인 흉수를 유발한다. 
폐렴의 조기진단은 완치적 치료를 보장하고 생존율을 높이는데 중요하다.
흉부 xray영상은 폐렴 진단에 가장 많이 사용되는 방법이다. 하지만 흉부xray검사는 어려운 작업이며 주관적인 가변성이 있다.
따라서 본 연구에서는 흉부 xray를 이용한 자동 폐렴 진단을 위한 컴퓨터 보조 진단 시스템을 개발하였다.
데이터 부족을 처리하기위해 심층 전이 학습 사용했고, GoogLeNet, ResNet-18 및 DenseNet121의 세 가지 합성곱 신경망 모델의 앙상블을 설계했다.
기본 학습자에게 할당된 가중치는 새로운 접근 방식을 사용해 결정했다.

## Intro
폐렴은 박테리아,바이러스 또는 진균류에 의해 유발될 수 있는 급성 폐 감염으로 폐를 감염시켜 기낭의 염증 및 흉막삼출액을 유발하며, 이는 폐에 액체가 차 있는 상태이다. 5세 미만 어린이 사망의 15%이상을 차지한다.
조기진단 및 관리는 질병이 치명적이지 않도록 예방하는데 중추적인 역할을 할 수 있다.
CAD(Computer Aided Diagnosis)시스템 개발 : 정확한 분류를 위한 심층 전이 학습 모델 앙상블

의료관련 이미지 분류 문제의 경우 방대한 양의 라벨링된 데이터는 전문의사가 각 이미지를 분류해야하므로 비용,시간 소모가 크다. 전이학습은 이를 극복하기 위한 해결방법이다.
큰 데이터 세트에서 훈련된 모델을 재사용하고 이 모델에서 결정된 네트워크 가중치를 적용한다.
ImageNet과 같은 대규모 데이터세트에서 훈련된 CNN모델은 1400만개 이상 이미지로 구성되어 있고, 생물 의학 이미지 분류 작업에 자주 사용된다.

-------------------------------------

폐렴은 호흡기 질환중 사망자 수 1위(대한민국 통계청,2017)에 해당하는 질병이며 흉부 엑스레이 촬영은 폐렴을 진단하기 위한 일차적인 검사 방법이다. / 의료인력이 부족함 / 의료영상을 전문적으로 판독할 수 있는 인력 양성에는 보통 10년 이상의 투자가 필요함 / 의료인력들의 피로감 누적에 따라 의료인력의 붕괴를 막는 조치가 필요
의료영상을 처리하는 딥러닝 모델을 통해 폐렴 진단 및 분석 작업에 컴퓨터의 판단을 개입시킨다면 최첨단 장비나 많은 의료인력이 없는 소규모 병원이나 진료소에서도 딥러닝 기술을 바탕으로 신속한 병변파악을 할 수 있어 의료진 인력 부족 현상의 해소에 이바지할 수 있음

**Comment by kikiru328** [comment]
[journal.pone.0256630.pdf](https://github.com/Pleasant-riot/Pneumonia-Detection/files/7556720/journal.pone.0256630.pdf)

**Reply by SHEYCHAE** [reply]
Pneumonia detection in chest X-ray images
using an ensemble of deep learning models

**Comment by kikiru328** [comment]
[electronics-10-01512-v2.pdf](https://github.com/Pleasant-riot/Pneumonia-Detection/files/7556721/electronics-10-01512-v2.pdf)

**Reply by SHEYCHAE** [reply]
Pneumonia Detection from Chest X-ray Images Based on
Convolutional Neural Network

**Comment by kikiru328** [comment]
[73466.pdf](https://github.com/Pleasant-riot/Pneumonia-Detection/files/7556725/73466.pdf)

**Reply by SHEYCHAE** [reply]
Models of Learning to Classify X-ray Images for the Detection of
Pneumonia using Neural Networks

**Comment by kikiru328** [comment]
https://link.springer.com/article/10.1007/s11554-021-01086-y

**Reply by SHEYCHAE** [reply]
A new approach for the detection of pneumonia in children using CXR images based on an real-time IoT system

**Comment by kikiru328** [comment]
https://scienceon.kisti.re.kr/commons/util/originalView.do?cn=JAKO202007862563265&oCn=JAKO202007862563265&dbt=JAKO&journal=NJOU00422586

**Reply by SHEYCHAE** [reply]
Evaluation of Classification and Accuracy in Chest X-ray Images using
Deep Learning with Convolution Neural Network

**Reply by SHEYCHAE** [reply]
Evaluation of Classification and Accuracy in Chest X-ray Images using
Deep Learning with Convolution Neural Network

**Comment by kikiru328** [comment]
https://ichi.pro/ko/cnneul-iyonghan-pyelyeom-jindan-823947836421

**Reply by SHEYCHAE** [reply]
CNN을 이용한 폐렴 진단

**Comment by kikiru328** [comment]
폐렴이란?

기침,염증 물질의 배출에 의한 가래, 숨쉬는 기능의 장애에 의한 호흡곤란 등 폐의 정상적인 기능에 장애가 생기는 폐 증상과, 구역, 구토, 설사 등의 소화기 증상 및 두통, 피로감, 근육통,관절통 등의 신체 전반에 걸친 전신 질환이 발생할 수 있다.

증상

폐렴은 폐에 염증이 생겨서 폐의 정상적인 기능에 장애가 생겨 발생하는 폐 증상과 신체 전반에 걸친 전신적인 증상이 나타난다. 폐 증상으로는 호흡기계 자극에 의한 기침,염증 물질의 배출에 의한 가래, 숨쉬는 기능의 장애에 의한 호흡곤란 등이 나타난다. 가래는 끈적하고 고름 같은 모양으로 나올수있고,피가 묻어나오기도 한다.

치료

원인균에 따른 치료를 진행
항생제 이용
중증인경우 적절한 항생제를 쓰더라도 사망가능성 높음
스스로 호흡이 불가능한 경우 인공호흡기에 의지해서 치료
그냥 항생제 쓰란다
치료기간은 2주

합병증

폐렴이 진행하여 패혈증이나 쇼크가 발생할 수 있고, 폐의 부분적인 합병증으로 기류나 기흉, 폐농양 등이 동반될 수 있다.

결론

폐렴이 뭘까?
바이러스나 세균으로 인한 염증 발생

폐렴 검사를 왜 하는지?
방치하면 합병증생김

데이터 구성은 어떻게 되어있는지?
캐글 ㄱㄱ

**Comment by kikiru328** [comment]
https://blog.goodaudience.com/yolo-object-detection-walkthrough-for-the-rsna-pneumonia-detection-challenge-123ec9a9adf2?gi=286cdcf26cbf

**Reply by SHEYCHAE** [reply]
YOLO Object Detection Walkthrough for the RSNA Pneumonia Detection Challenge

**Reply by kikiru328** [reply]
# RNSA 폐렴 검사 (캐글 데이터 설명)
## 정상 폐
정상 폐는 공기로 가득차 있음. x-ray 폐 부분이 검은색으로 채워져 있으면 정상임.
![image](https://user-images.githubusercontent.com/60537388/142753813-0f180992-36cc-4853-8e45-fe6dcab09f46.png)

## 비정상 폐 (폐렴)
폐렴이 걸려 있을시 공기대신 액체, 세균, 면역세포 등 다양한 것으로 채워져 있음. 이것들은 회색과 비슷한 색으로 불투명하게 나와있음. 흑백보단 구름과 같은 형태로 불투명함.
![image](https://user-images.githubusercontent.com/60537388/142753866-ebf936fb-9451-45bd-9f4e-5e862094a5d0.png)

# 중요한 점
폐가 불투명하다고 무조건 폐렴과 관계된 것은 또 아니다.  아래와 같은 사진들은 Not Normal / No Lung Opacity로 폐렴과 관계되진 않지만 비정상으로 분류가 된다. 
![image](https://user-images.githubusercontent.com/60537388/142753913-6f91151e-7484-4ab5-b74a-28e95c74a4ea.png)

# 그래서 가장 중요한 점은 무엇인가?
Lung Opacity, 폐렴 환자는 분류가 가능하지만,  폐렴이 아닌 No Lung Opacity / Not Normal 인 사람들과 비교하는 것이 가장 큰 [도전과제]

**Comment by SHEYCHAE** [comment]
[잔류_밀집_네트워크_기반_영.pdf](https://github.com/Pleasant-riot/Pneumonia-Detection/files/7575888/_._._._.pdf)
전처리

**Comment by SHEYCHAE** [comment]
[심포지움  폐렴 ; 폐렴의 진단.pdf](https://github.com/Pleasant-riot/Pneumonia-Detection/files/7575890/default.pdf)

**Comment by SHEYCHAE** [comment]
X-ray 이미지 분류 및 모델 평가
https://ichi.pro/ko/x-ray-imiji-bunlyu-mich-model-pyeong-ga-132010186903541

**Comment by kikiru328** [comment]
FPN? P-Yolo * ( Pneumonia Yolo
[An_Improved_Algorithm_for_Detecting_Pneumonia_Base.pdf](https://github.com/Pleasant-riot/Pneumonia-Detection/files/7575934/An_Improved_Algorithm_for_Detecting_Pneumonia_Base.pdf)
)

**Comment by kikiru328** [comment]
https://cs230.stanford.edu/projects_fall_2018/reports/12449174.pdf

**Comment by kikiru328** [comment]
https://link.springer.com/content/pdf/10.1007/s13369-021-06127-z.pdf

**Comment by kikiru328** [comment]
https://arxiv.org/pdf/1904.00937.pdf

**Comment by KIBEOMP** [comment]
![image](https://user-images.githubusercontent.com/86215240/143383512-bc587045-2a83-42d2-b260-58caa513bdb7.png)

[정상흉부 x-ray 판별위치도.pdf](https://github.com/Pleasant-riot/Pneumonia-Detection/files/7600365/x-ray.pdf)
